### ISSUE TITLE
Make casing of settings titles / text consistent

### ISSUE THREAD

**Issue by pes10k** [brave/brave-browser#8822]
the follow are all cases that don't follow sentence casing in brave://settings

```
- Side menu -> "Help Tips"
- Appearance -> Themes -> "Open Web Store"
- Appearance -> "Always show bookmarks on New Tab page"
- Appearance -> "Pressing Tab on a webpage"
- Shields -> "Look & Feel"
- Extensions -> "Web3 provider for using Dapps"
- Extensions -> "Private Window with Tor"
- Extensions -> Get more extensions -> "Open Web Store"
- Privacy and security -> "WebRTC IP Handling Policy"
- Privacy and security -> "Use Google Services for Push Messaging"
```

cc: @karenkliu @rebron

**Comment by edwardqiu** [comment]
Hey, I would love to give this issue a try!

Just to double check, we expect the following changes on the `brave://settings` page:

| Text Before | Text After | Location |
|---|---|---|
| Help Tips | Help tips  | Side menu under `settings-menu` | 
| Open Web Store  | Open web store  | under `Appearance` section |
| Always show bookmarks on New Tab page | Always show bookmarks on new tab page | under `Appearance` section | 
| Pressing Tab on a webpage highlights links, as well as form fields | Pressing tab on a webpage highlights links, as well as form fields | under `Appearance` only on macOS | 
| Look & Feel | Look & feel | under `Shields` section | 
| Web3 provider for using Dapps | Web3 provider for using dapps | under `Extensions` section | 
| Private Window with Tor | Private window with Tor | under `Extensions` section | 
| Open Web Store | Open web store  | description of `Get more extensions` under `Extensions` section | 
| WebRTC IP Handling Policy | WebRTC IP handling policy | under `Privacy and Security` section | 
| Use Google Services for Push Messaging | Use Google services for push messaging | under `Privacy and Security` section |

**Comment by rebron** [comment]
@edwardqiu Keep `Dapps` the same. Everything else looks good.

**Comment by edwardqiu** [comment]
Hey, @rebron. Since this is my first time contributing here, it took me a while to get my dev environment setup. 

I've made the changes [here](https://github.com/edwardqiu/brave-core/commit/c079178862a285ec76bef7afee99860c7414700c). I also tested all of them manually on Ubuntu 16.04, except for the following changes:
`Appearance -> Themes -> "Open Web Store"`, because the text change is not reached because of [this code block](https://github.com/edwardqiu/brave-core/blob/c079178862a285ec76bef7afee99860c7414700c/app/settings_strings.grdp#L815-L828).
`Appearance -> "Pressing Tab on a webpage"`, because this text change is also not reached because it is within [this code block](https://github.com/edwardqiu/brave-core/blob/c079178862a285ec76bef7afee99860c7414700c/app/settings_strings.grdp#L852-L858).

For the exceptions, would there be someone with macOS be able to checkout my branch and test it?

**Comment by edwardqiu** [comment]
I've also noticed on Linux - there are a few more things that may not be following sentence casing. Would you like these things to be changed to follow sentence casing as well?

![image](https://user-images.githubusercontent.com/14011954/82639031-c76a2280-9bbc-11ea-8256-5d9b7cb175ad.png)
![image](https://user-images.githubusercontent.com/14011954/82639054-d2bd4e00-9bbc-11ea-9e4f-e00c6330f287.png)
![image](https://user-images.githubusercontent.com/14011954/82639122-f2ed0d00-9bbc-11ea-897c-c2e557c17e69.png)
![image](https://user-images.githubusercontent.com/14011954/82639166-0b5d2780-9bbd-11ea-8766-5fdd5c1d48a9.png)

**Comment by jonathanKingston** [comment]
![image](https://user-images.githubusercontent.com/338988/94862093-66870c80-0430-11eb-9169-19501d51d48c.png)
I noticed this in the mentioned text changes the & is being escaped in: Version 1.9.72 Chromium: 81.0.4044.138 (Official Build) (64-bit)

**Comment by bsclifton** [comment]
@jonathanKingston that's a pretty old version - I think we've fixed it since then. Can you please update Brave and report back if it's fixed?

**Comment by jonathanKingston** [comment]
Ah so it is :') thanks, I thought I was on the auto update path. thanks!

**Comment by bsclifton** [comment]
@jonathanKingston no worries - got me noticing https://github.com/brave/brave-core/pull/5631 which somehow got past the radar (so many PRs!). Saw that got approved- just needs a rebase. Will try to sort that out

**Comment by btlechowski** [comment]
Verification passed on

Brave | 1.17.32 Chromium: 86.0.4240.80 (Official Build) nightly (64-bit)
-- | --
Revision | 7ed88b53bda45a2d19efb4f8706dd6b6cad0d3af-refs/branch-heads/4240@{#1183}
OS | Ubuntu 18.04 LTS

Verified test plan from the description

Side menu -> "Help Tips" to "Help tips"
![image](https://user-images.githubusercontent.com/34715963/96031796-1e1f1400-0e5e-11eb-81b0-748bb24b2f3d.png)

Appearance -> Themes -> "Open Web Store" to "Open web store"
Not available on Ubuntu 18.04. It has `Classic` on clean install

Appearance -> "Always show bookmarks on New Tab page" to "Always show bookmarks on New Tab page"
![image](https://user-images.githubusercontent.com/34715963/96032813-84f0fd00-0e5f-11eb-9c5b-ed1765d8da91.png)

Appearance -> "Pressing Tab on a webpage" to "Press tab on a webpage"
Not available on Ubuntu 18.04

Shields -> "Look & Feel" to "Look & feel"
![image](https://user-images.githubusercontent.com/34715963/96034293-96d39f80-0e61-11eb-91ae-17c34dffe35d.png)

Extensions -> "Private Window with Tor" to "Private window with Tor"
![image](https://user-images.githubusercontent.com/34715963/96034856-7821d880-0e62-11eb-82f9-187533350526.png)

Extensions -> Get more extensions -> "Open Web Store" to "Open web store"
![image](https://user-images.githubusercontent.com/34715963/96034951-98519780-0e62-11eb-8bba-c81968a2ceaa.png)


Privacy and security -> "WebRTC IP Handling Policy" to "WebRTC IP handling policy"
![image](https://user-images.githubusercontent.com/34715963/96035019-af908500-0e62-11eb-92b1-3e115209b1a8.png)

Privacy and security -> "Use Google Services for Push Messaging" to "Use Google services for push messaging"
![image](https://user-images.githubusercontent.com/34715963/96035153-e9fa2200-0e62-11eb-847c-4466ffc1dd53.png)

Logged https://github.com/brave/brave-browser/issues/12147 for missing translations
